network/kademlia_test.go

```diff
@@ -65,6 +67,24 @@ func (tk *testKademlia) newTestKadPeer(s string) *Peer {
 	return NewPeer(&BzzPeer{BzzAddr: testKadPeerAddr(s)}, tk.Kademlia)
 }
 
+func addressGenerator(numAddress int, base string) []string {
```

Comment:
I can't see this being used in this delta. Besides, address generating code already exists. If this is indeed needed, then please add a comment describing the reason why.

Reply:
Forgot to remove